exercises/09-documentation.md

```diff
@@ -29,83 +29,36 @@ restrictions, possible return values, expected arguments, etc.
 
 ### Overview
 
-- Add documentation to keywords, which need to be run inside iframes
 - Add documentation to your `Change Important Number` keyword, which tells the difference in
 running with `execute_javascript=${TRUE}` or with `${FALSE}`.
+- Add proper tags to your test case.
 
 ### Step-by-step
 
 <details>
-  <summary>Browser</summary>
+  <summary>Add meaningul documentation.</summary>
 
-**Add meaningful documentation to your keywords.**
+<br />
 
-We've only added a few keyword, but we already have some special clauses,
-which we need to know when calling our keywords. For example, our
-`Fill Form With Valid Data` keyword calls `Fill All Form Fields` (and all
-keywords that calls) inside an iframe. However, the latter keywords
-have no notion that they _must_ be called inside an iframe in order to work.
-That's something we will probably forget in the future and will confuse new
-teammates, se let's add that quickly to our keywords.
-
-Following a logical order, let's add the `[Documentation]` right to the
-beginning of the keyword.
-
-- Add a notion to `Fill All Form Fields`, `Fill Form Field`, `Select Date From Future`,
-and `Change Important Number` that they must be called inside an iframe to work.
-
-There doesn't seem to be any other restrictions to our keywords. However,
+Keyword and test case documentation shouldn't just state the obvious. However,
```

Comment:
shouldn't just state -> shouldn't be added just to state the obvious
However, in case of special requirements and nontrivial use cases, there should be a documented explanation.
^ suggestion for clarity